src/Options.php

```diff
@@ -179,23 +179,23 @@ public function setMbDetectOrder($detectOrder): void
     }
 
     /**
-     * Gets whether the stacktrace must be auto-filled.
+     * Gets whether the stacktrace will be attached on captureMessage.
```

Comment:
Is this option specifically designed to work only in the `captureMessage` method? It seems a bit strange...